plat/xen/arm/arch_events.c

```diff
@@ -39,12 +44,28 @@ static void virq_debug(evtchn_port_t port __unused,
 
 static evtchn_port_t debug_port = -1;
 
+static int arch_events_irq_handler(void *args __unused)
+{
+	ukplat_lcpu_irqs_handle_pending();
+	return 1;
+}
+
 void arch_init_events(void)
 {
+	int rc;
+
 	debug_port = bind_virq(VIRQ_DEBUG, (evtchn_handler_t)virq_debug, 0);
 	if ((int) debug_port == -1)
 		UK_CRASH("Failed to initialize events\n");
 	unmask_evtchn(debug_port);
+
+	/* Register the lcpu_run and lcpu_wakeup interrupt handlers */
```

Comment:
This comment seems wrong, maybe `Register the the events interrupt handler`?

Reply:
fixed